UPGRADE-6.8.md

```diff
@@ -10,6 +10,23 @@
 
 # Storefront
 
+## Removed `category_url` and `category_linknewtab` twig functions
```

Comment:
this is wrong right?

Reply:
it was not part of the original PR, yes. but it is already in the 6.8 upgrade guide on trunk, so I thought it won't hurt, if this is backported to 6.6 :man_shrugging: